src/event.js

```diff
@@ -610,8 +610,7 @@ jQuery.event = {
 			event,
 			{
 				type: type,
-				isSimulated: true,
-				originalEvent: {}
```

Comment:
> As far as docs, the page for .on() already says that for native events there's an originalEvent property.

I see only 

> This object is a normalized subset of data provided by the browser; the browser's unmodified native event object is available in event.originalEvent.

Don't see "native" there, not sure if that phrase suppose to warn user about simulated event. In any case, focus(in | out) do exist in most browsers and in IE it would exist for real. User might not even know we shim it.

> but degrades event performance, as @gibson042 pointed out.

I doubt it, event processing is pretty hot already, event creation should not reflect on it.